tests/JsonTest.php

```diff
@@ -11,35 +13,35 @@
 
 class JsonTest extends TestCase
 {
-    public function testParseNull()
+    public function testParseNull(): void
     {
         $this->assertNull(Json::parse(null));
     }
 
-    public function testParseValid()
+    public function testParseValid(): void
     {
-        $this->assertEquals(['foo' => 'bar'], Json::parse('{"foo": "bar"}'));
+        $this->assertSame(['foo' => 'bar'], Json::parse('{"foo": "bar"}'));
     }
 
-    public function testParseErrorEmptyString()
+    public function testParseErrorEmptyString(): void
     {
         $this->expectParseException('', 0);
     }
 
-    public function testParseErrorObjectEmptyString()
+    public function testParseErrorObjectEmptyString(): void
     {
         $this->expectParseException(new TestStringable(''), 0);
```

Comment:
The `new TestStringable('')` should change into the string type, not passing the object type.